.circleci/config.yml

```diff
@@ -139,6 +139,9 @@ jobs:
       - run:
           name: 'Run Super Scaffolding Test'
           command: "bundle exec rails test test/system/super_scaffolding_test.rb"
+      - run:
+          name: 'Run Super Scaffolding Partial Test'
+          command: 'bundle exec rails test test/system/super_scaffolding_partial_test.rb'
```

Comment:
I wonder if we should refactor these tests into a `test/system/super_scaffolding` directory on the starter repo end? So we could just get `bundle exec rails test test/system/super_scaffolding/` here.

Could do `test/system/super_scaffolding/super_scaffolding_test.rb` and `test/system/super_scaffolding/partial_test.rb` for the test names.

Reply:
@kaspth That sounds reasonable.

Reply:
@jagthedrummer do you want to look at doing it? I'm kinda up to my ears in open PRs right now.

Reply:
@kaspth I'll open an issue for it and we can do it later. I'm also swamped with open stuff (which I'm trying to get closed). What we have currently is good enough for now.

Reply:
https://github.com/bullet-train-co/bullet_train/issues/833